tests/post_training/data/wc_reference_data_2024.5.yaml

```diff
@@ -3,12 +3,8 @@ tinyllama_data_aware_gptq_backend_OV:
   num_int4: 94
   num_int8: 124
   metrics_xfail_reason: "Issue-148819"
-tinyllama_scale_estimation_per_channel_backend_OV:
-  metric_value: 0.80798
-  num_int4: 188
-  num_int8: 124
-  metrics_xfail_reason: "Issue-148819"
 tinyllama_NF4_scale_estimation_stateful_per_channel_backend_OV:
   metric_value: 0.87132
   num_int4: 11
   num_int8: 290
+  metrics_xfail_reason: "Issue-148819"
```

Comment:
Why did you add xfail for tinyllama_NF4_scale_estimation_stateful_per_channel_backend_OV?

Reply:
Good catch! I've removed it.